icu4c/source/i18n/double-conversion.cpp

```diff
@@ -432,21 +433,55 @@ void DoubleToStringConverter::DoubleToAscii(double v,
 }
 
 
+namespace {
+
+inline char ToLower(char ch) {
+  static const std::ctype<char>& cType =
+      std::use_facet<std::ctype<char> >(std::locale::classic());
+  return cType.tolower(ch);
```

Comment:
I'm not sure if this functionality in double-conversion is actually used by ICU? (Maybe it could be removed?)

If it is used, then I have a few thoughts/comments:
At least with MSVC, `std::locale` is generally rather bloated and slow. :(
Since this is always using the "classic" C locale anyways, could we just use `uprv_invCharToLowercaseAscii` instead?
(This would also presumably get rid of the two new `depstest` failures as well).

Reply:
Looks like it's not used.  I #if'd it out, similar to what I've done elsewhere in this file.